libs/blocks/chart/chart.js

```diff
@@ -116,28 +150,43 @@ export const getChartOptions = (chartType, dataset, colors, unit = '') => {
 };
 
 const init = async (el) => {
+  const parent = el.parentElement;
   const children = el?.querySelectorAll(':scope > div');
   children[0]?.classList.add('chart_title');
   children[1]?.classList.add('chart_subTitle');
   children[3]?.classList.add('chart_footnote');
 
-  // ToDo: Replace hardcoded size MWPW-112994
-  const size = LARGE;
+  // Add chart container
+  const container = document.createElement('div');
+  container.className = 'chart_container';
+  container.append(...children);
+  el.appendChild(container);
+
+  // Grab section metadata
+  const chartStyles = parent.className.split(' ').slice(1);
```

Comment:
`element.classList` provides classes as an array

Reply:
I was getting an issue earlier when using the `find()` method few lines down, but have fixed it. Thanks